cmd/datasets/push.go

```diff
@@ -81,3 +89,43 @@ func getDatasetFiles(datasetConfig *config.DatasetConfig) ([]string, error) {
 	}
 	return allMatchedFiles, nil
 }
+
+func addDatasetIfNotExisted(ctx context.Context, datasetConfig *config.DatasetConfig) error {
+	datasets, _, err := ApiClient.GetDatasets(ctx).Execute()
+	if err != nil {
+		return fmt.Errorf("Failed to get datasets: %v", err)
```

Comment:
use logger class for this as well?

Reply:
same for the other usages, I wont comment on each

Reply:
fmt.Errorf not Print error (It create a error)